uint8 gait_cycles
---
foothold_planner_msgs/GlobalFootholds footholds